Assignment 5/SenerioOne.js

```diff
@@ -0,0 +1,34 @@
+try 
+{
+  function CarAccidentByWoman() 
```

Comment:
You can define function outside of the try cathc block. Then you call function in try catch. This is more suitable